CMakeLists.txt

```diff
@@ -79,6 +79,12 @@ option(sctp_sanitizer_memory "Compile with memory sanitizer" 0)
 
 option(sctp_build_fuzzer "Compile in clang fuzzing mode" 0)
 
+option(sctp_use_lwip "build with lwip" ON)
```

Comment:
Please don't include LWIP by default.

Reply:
Sure, I will disable it when it is merged into it officially. Alright, it is one very rough version to show the idea.